fast_build_and_deploy_docs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+latesttag=$(git describe --tags `git rev-list --tags --max-count=1`)
+echo checking out ${latesttag}
+git checkout ${latesttag}
+git submodule update --remote
+pushd docs/source
+git checkout
+make html
+ghp-import -c docs.pymc.io -n -p _build/html/
+popd
+cd docs/source/pymc-examples
+echo checking out pymc-examples fast build branch
+git checkout fast-docs-build
```

Comment:
this would need doing before `make clean` and `make html`

Reply:
Sorry, I went a bit overboard and kept pushing to this PR. I have made the suggested changes, and managed to run the script successfully. I am getting some warnings but it looks like it is building now, still take ~1 minute to complete.